Makefile

```diff
@@ -29,9 +29,12 @@ build: ## Build/compile the OpenFGA service
 run: build ## Run the OpenFGA server with in-memory storage
 	./bin/openfga run
 
+.PHONY: migrate-postgres
+migrate-postgres: build
+	./bin/openfga migrate --datastore-engine postgres --datastore-uri 'postgres://postgres:password@localhost:5432/postgres?sslmode=disable'
+
 .PHONY: run-postgres
-run-postgres: build ## Run the OpenFGA server with Postgres
-	# nosemgrep: detected-username-and-password-in-uri
```

Comment:
Why did you remove the `nosemgrep` comment 😭

Reply:
You can move the URI to a variable and then put the comment above it, I suppose

Reply:
Whoops. That must have happened when I merged main into my branch. Will fix it. Good catch. Thanks!

Reply:
And now I realized that the `run-postgres` command will no longer work since we don't have those env vars anymore.